src/main/kotlin/ContaBancaria.kt

```diff
@@ -1,4 +1,7 @@
 package org.example
+import java.time.LocalDateTime
+import java.util.Scanner
+import java.util.UUID
 
 //' Desenvolver um sistema bancário que permita a criação de contas bancárias e depósitos.
```

Comment:
Ajustar comentários:

```suggestion
/**
 * Desenvolver um sistema bancário que permita a criação de contas bancárias e depósitos.
 * 
 * Regras:
 * - O saldo da conta bancária deve ser atualizado após cada depósito.
 * - O sistema deve permitir a criação de depósitos em uma conta bancária.
 * - O sistema deve permitir a criação de contas bancárias.
 * - O sistema deve permitir a criação de usuários.
 * - O sistema deve permitir a associação de um usuário a uma conta bancária.
 * - O sistema deve permitir a consulta de depósitos realizados em uma conta bancária.
 * - Gerar recibo de depósito.
 * - Validar se os dados da conta bancária (número da conta e agência) são válidos;
 * - Garantir que o valor do depósito seja um número positivo;
 * - O sistema deve atualizar o saldo da conta automaticamente após um depósito bem-sucedido.
 * - Gerar recibo Número da conta e agência (com informações parcialmente ocultas, para segurança), Valor do depósito, Data e hora da transação.
 */
```